mod/research/set/self/research_page.rb

```diff
@@ -15,6 +15,14 @@
     _render_core
   end
 
+  view :add_relation, cache: :never do
+    @answer_view = :research_form
+    @answer_card = Card.new name: [metric, company, year.to_s],
+                            type_id: RelationshipAnswerID
+    @answer_card.define_singleton_method(:unknown?) { true }
```

Comment:
I gather `new?` won't work?

Reply:
(maybe worth a comment?)

Reply:
That's one of the many routes I tried. Could be that it's no longer necessary. I have to check.